packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -309,4 +309,93 @@ describeNoCompat("GC DataStore Tombstoned When It Is Sweep Ready", (getTestObjec
         assert(closeError.errorType === "dataCorruptionError");
         assert(closeError.message === "Context is tombstoned: Call site -  process!");
     });
+
+    // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+    itExpects("Container loaded after sweep timeout expires is unable to load tombstoned datastores",
+    [
+        {
+            eventName: "fluid:telemetry:Summarizer:Running:SweepReadyObject_Loaded",
+        },
+    ],
+    async () => {
+        const { testDataObjectId, summaryVersion } = await getUnreferencedDataStoreIdAndSummaryVersion();
+
+        // Wait some time, the datastore should not be sweep ready after this wait
+        await delay(sweepTimeoutMs);
+
+        // Load a new container and summarizer based on the latest summary, summarize
+        const {
+            container: summarizingContainer2,
+            summarizer: summarizer2,
+        } = await loadSummarizerAndContainer(summaryVersion);
+
+        // Use the request pattern to get the testDataObject - this is unsafe and no one should do this in their
+        // production application - causes an inactive loaded and changed error
+        await assert.doesNotReject(
+            async () => requestFluidObject<ITestDataObject>(summarizingContainer2, testDataObjectId),
+            `Should be able to still request unreferenced datastores`,
```

Comment:
Is this because of that corner case where the Summarizer doesn't initialize the Trackers on connect?  Not sure it's worth highlighting so much in the "script" of the test.  I would ditch the `assert.doesNotReject` and just call `requestFluidObject`.  If it rejects it'll fail the test.

If you really do want to draw attention to the fact that this succeeds, leave it, but maybe give more explanation why, because it's pretty surprising (since we already waited `sweepTimeoutMs`).

Reply:
OK, I'll make it a comment, I thought this was important enough to illustrate in the test as it was a corner case

Reply:
I just realized we already test this above so I'm going to remove this